src/screens/home-screen.tsx

```diff
@@ -2,23 +2,24 @@ import  React  from "react";
 import { SafeAreaView, ActivityIndicator } from "react-native";
 import { UsersList } from "../components/users-list";
 import { useGetUsers } from "../hooks/use-get-users";
+import { NavigationFAB } from "../components/navigation-fab";
 
-export function HomeScreen(): JSX.Element {
+interface Props {
```

Comment:
```suggestion
interface HomeScreenProps {
```

Mais uma convenção 🤣 
Nome da interface de props -> Mesmo nome do componente seguido do sufixo `Props`